redirects.yaml

```diff
@@ -719,3 +719,6 @@ usn/rss\.xml: "https://usn.ubuntu.com/rss.xml"
 (?P<page>.*)\.html: "/{page}"
 telco/?: "/telecommunications"
 telcos/?: "/telecommunications"
+fips: "/security/certifications"
+cc: "/security/certifications"
+cis: "/security/certifications"
```

Comment:
I think these need to have the /? added to the first half e.g. ```fips/?: "/security/certifications"```

also, they should be in alphabetical order

and you need to write something in the ## Done bullet.